docs/10-roadmap.md

```diff
@@ -4,18 +4,15 @@
 This page is still being developed. More content will be added soon!
 :::
 
-## Roadmap
 The project roadmap for Zarf is hosted on a [GitHub Project Board](https://github.com/orgs/defenseunicorns/projects/1). There, you can see what we're working on and what work we're prioritizing.
 
-If you want to add your own bug reports or feature requests to be considered on our roadmap, please feel free [add an issue](https://github.com/defenseunicorns/zarf/issues) on the GitHub repository.
+If you want to add bug reports or feature requests to be considered on our roadmap, please feel free to [add an issue](https://github.com/defenseunicorns/zarf/issues) on the GitHub repository.
 
-<br />
 
 ## GA Release
-Right now, Zarf is still in its 'beta' phase. We are working on a few final things before we release the official 1.0 general availability (GA) release. The current planned date for the GA release is mid June 2022. The work still needed for the GA release can be found on the roadmap with [this filter](https://github.com/orgs/defenseunicorns/projects/1/views/1?filterQuery=repo%3A%22defenseunicorns%2Fzarf%22+milestone%3A%22zarf+ga%22).
-
 
+Right now, Zarf is still in its 'beta' phase. We are working on a few final things before we release the official 1.0 general availability (GA) release. The work still needed for the GA release can be found on the roadmap with [this filter](https://github.com/orgs/defenseunicorns/projects/1/views/1?filterQuery=repo%3A%22defenseunicorns%2Fzarf%22+milestone%3A%22zarf+ga%22).
```

Comment:
This link does not take them to the information that the sentence implies @jeff-mccoy is there another place we want the user to go for updates to v1.0 or should we just list out the goals the team wants to meet before cutting v1.0 - currently documented in coda

Reply:
A decision was made with the team to move the roadmap to READ.me in a different issue. Will create a separate issue for updating the read me to include the road map and then make updating the linked part of that PR